deploy/operator.yaml

```diff
@@ -18,7 +18,14 @@ spec:
         node-role.kubernetes.io/master: ""
       priorityClassName: "system-cluster-critical"
       tolerations:
-      - operator: Exists
+      - key: "node.kubernetes.io/unreachable"
```

Comment:
Should these changes also be applied to the upstream manifests? Or are these tolerations only needed for OKD/OpenShift clusters?